Formula/mpw.rb

```diff
@@ -16,26 +16,24 @@ class Mpw < Formula
   end
 
   bottle do
-    sha256 cellar: :any, arm64_big_sur: "252fbad588409e4fd5be091f7c248150e75b995d39ce334938b9a7d33534cbc9"
-    sha256 cellar: :any, big_sur:       "4ae85b3d30e47294436b7fddca456c98ed2bf546793f2ef9d57a372d782fb072"
-    sha256 cellar: :any, catalina:      "2f275d762a9c73bd6b3f2e5a7f3f13a9c99ddfc3e2f89a2ededa07ba89b6de40"
-    sha256 cellar: :any, mojave:        "9103716223529cd3e2cb969e904892bf2022cb8e73918418f2d3d343d1325c80"
-    sha256 cellar: :any, high_sierra:   "07b89df8d96f9c1cebbf6296a4e98b2bac833c45f736b646a1eba24bd5244732"
+    cellar :any
   end
 
+  depends_on "jq" => :build
   depends_on "json-c"
   depends_on "libsodium"
   depends_on "ncurses"
 
   def install
+    cd "cli" unless build.head?
     cd "platform-independent/c/cli" if build.head?
 
-    ENV["targets"] = "mpw"
-    # not compatible with json-c 0.14 yet
-    ENV["mpw_json"] = "0"
+    ENV["targets"] = "mpw mpw-tests"
+    ENV["mpw_json"] = "1"
     ENV["mpw_color"] = "1"
 
     system "./build"
+    system "./mpw-tests"
     system "./mpw-cli-tests"
```

Comment:
If these are just integration tests too (i.e. not necessary for the build) we should probably skip these also.